package.json

```diff
@@ -62,6 +62,7 @@
     "@wordpress/block-library": "2.24.0",
     "babel-loader": "8.1.0",
     "browserify-transform-tools": "1.7.0",
+    "chokidar-cli": "^2.1.0",
```

Comment:
It feels off to me that we have Gulp for managing and watching tasks, but instead we're using a bare `.js` file that we're running via `node`, and now we're adding the Gulp watch dependency so we can configure that separately.

Can we not manage the Style Dictionary build script as a Gulp task and then the watch command from the same place as everything else?

Reply:
Sorry about that, I was under the impression that we were trying to move away from Gulp toward npm scripts when possible. I'll look into how we can use Gulp to manage this.